### PR TITLE
Ensure builder file exists in GCS before publishing manifest

### PR DESCRIPTION
**Pull request by nkubala**
This can now be run as a presubmit on config changes to make sure we don't push a bad config.

@dlorenc @sharifelgamal @duggelz

### CODE REVIEW

File: runtime_builders/builder_util.py [hunk 1]
```diff
@@ -169,6 +78,18 @@ def load_manifest_file():
         os.remove(tmp)
 
 
+def file_exists(remote_path):
+    try:
+        logging.info('Checking file {0}'.format(remote_path))
+        tmpdir = tempfile.mkdtemp()
+        tmp_file = os.path.join(tmpdir, 'tmp')
```
Comment (dlorenc) [inline runtime_builders/builder_util.py]: There's a 'gsutil stat' or a 'gsutil ls' command that wouldn't require copying the file locally.

File: runtime_builders/publish_builders.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 import sys
 
 import builder_util
+from verify_manifest import verify_manifest
```
Comment (dlorenc) [inline runtime_builders/publish_builders.py]: We typically import just the package and then reference stuff like verify_manifest.verify_manifest

File: runtime_builders/builder_util.py (hunk 1 above)
Comment (nkubala) [reply runtime_builders/builder_util.py]: updated

File: runtime_builders/publish_builders.py (hunk 2 above)
Comment (nkubala) [reply runtime_builders/publish_builders.py]: fixed

File: runtime_builders/builder_util.py [hunk 3]
```diff
@@ -169,6 +77,21 @@ def load_manifest_file():
         os.remove(tmp)
 
 
+def file_exists(remote_path):
+    try:
+        logging.info('Checking file {0}'.format(remote_path))
+        command = ['gsutil', 'stat', remote_path]
+        proc = subprocess.Popen(command, stdout=subprocess.PIPE,
```
Comment (dlorenc) [inline runtime_builders/builder_util.py]: Would subprocess.check_call work here?

Comment (nkubala) [reply runtime_builders/builder_util.py]: yep, changed

**Comment by duggelz** [comment]
LGTM except unit test would be nice.

**Pull request merged**